github3/repos/branch.py

```diff
@@ -372,19 +388,145 @@ def users(self, number=-1):
         from .. import users
         return self._iter(int(number), self.users_url, users.ShortUser)
 
+    @decorators.requires_auth
+    def add_teams(self, teams):
+        """Add teams to the protected branch.
```

Comment:
Missing a newline between this line and "See"

Reply:
Fixed